scss/_button-group.scss

```diff
@@ -73,88 +74,50 @@
 //
 
 .dropdown-toggle-split {
-  padding-right: subtract($dropdown-padding-x, $border-width);
-  padding-left: subtract($dropdown-padding-x, $border-width);
-  border-color: $black;
+  padding-right: subtract($dropdown-padding-x, $btn-border-width); // Boosted mod
```

Comment:
For a Boosted mod on multiline, please use:

// Boosted mod:
<..>
// End mod